bioconda.py

```diff
@@ -40,10 +44,11 @@ def parse_meta_dot_yaml_for_source_link(filepath):
         str - the source code link (e.g. GitHub) for the package
 
     """
-    link = ""
-    with open(filepath, "r") as meta_file:
-        for line in meta_file.readlines():
-            if "home" in line:
-                link = clean_github_link(line)
+    # Strip fname and keep path
+    path = os.path.dirname(filepath)
+    # This unusual call is a quirk of bioconda_utils.recipe
+    recipe = Recipe.from_file(path, path)
+    # Get link[s]
+    links = list(nested_dictionary_extract("url", recipe.meta))
```

Comment:
Why is `clean_github_link` not needed here?

Reply:
Same. And feel free to revise ```clean_github_link``` if it needs to be revised.

Reply:
Why not have ```nested_dictionary_extract``` return a list? I suggest this only because I see the code currently immediately converts the generator returned from ```nested_dictionary_extract``` into a list.

Reply:
It was an oversight on my part. Good catch. Using clean_github_link.

Reply:
> Why not have `nested_dictionary_extract` return a list? I suggest this only because I see the code currently immediately converts the generator returned from `nested_dictionary_extract` into a list.

I thought that using a generator for `nested_dictionary_extract` was a little more elegant, since it isn't reliant on memory and it's a little shorter and more readable than a list version would be. Might be overkill.

Reply:
On the ```nested_dictionary_extract``` point: roger. If it works for you, it works for me. Please proceed as you propose.

Reply:
Note that the list may contain a single element which is a list of urls. Handled with an `isinstance()` test. Is this what is expected?